packages/components/src/snackbar/README.md

```diff
@@ -36,17 +36,52 @@ const MySnackbarNotice = () => (
 );
 ```
 
-#### Props
+### Props
```

Comment:
Could we sort the props alphabetically?

Reply:
Sure! Committed in https://github.com/WordPress/gutenberg/pull/45472/commits/84071c6354a66ce487477d632b64ee3e14cdf5c7